packages/datatrak-web/src/features/Questions/EntityQuestion/EntityQuestion.tsx

```diff
@@ -33,14 +33,15 @@ const Label = styled(Typography).attrs({
 `;
 
 const useSearchResults = (searchValue, config) => {
-  const filters = useEntityBaseFilters(config);
+  const filter = useEntityBaseFilters(config);
```

Comment:
Could we re-work `useEntityBaseFilters` so that it filters by attributes on the `entity-api`? Rather than doing the filtering on the frontend

Reply:
Heya, I believe it does do the filtering on the API, but this hook is just generating the filter to snd to the api from other answers

Reply:
Really? It looks like `useAttributeFilter` is doing the attribute filtering?